packages/utilities/moment-timezone-include/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/moment-timezone-include",
```

Comment:
Need to update moment-timezone-include package lock. Or tbh, this doesnt need changing

Reply:
yup, this was a mindless update